apis/memcached/v1beta1/webhook_suite_test.go

```diff
@@ -31,22 +31,28 @@ import (
 	admissionv1beta1 "k8s.io/api/admission/v1beta1"
 	//+kubebuilder:scaffold:imports
 	"k8s.io/apimachinery/pkg/runtime"
+	kscheme "k8s.io/client-go/kubernetes/scheme"
 	"k8s.io/client-go/rest"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/envtest"
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+
+	metricsserver "sigs.k8s.io/controller-runtime/pkg/metrics/server"
```

Comment:
the kuttl error referring to this line. But v0.17 has this package available https://github.com/kubernetes-sigs/controller-runtime/tree/v0.17.0/pkg/metrics/server . Based on the error message for some reason go is pulling the version of the controller runtime back from 0.17 to 0.14.6. The only dependency that I can think of doing this is lib-common: https://github.com/openstack-k8s-operators/lib-common/blob/main/modules/common/go.mod#L18 So I suggest to try to bump the runtime in lib-common first

Reply:
openstack-operator has a replaces for this:
https://github.com/openstack-k8s-operators/openstack-operator/blob/main/go.mod#L125